educe/stac/learning/cmd/addressee.py

```diff
@@ -1,97 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# Author: Eric Kow
-# License: CeCILL-B (French BSD3)
-
-"""
-Guess the addresee for each EDU
-"""
-
-from __future__ import print_function
-
-from educe.stac import postag, corenlp
-from educe.stac.annotation import is_edu
-from educe.stac.context import sorted_first_widest
-from educe.util import\
-    add_corpus_filters, fields_without, mk_is_interesting
-import educe.corpus
-import educe.glozz
-import educe.learning.keys
-import educe.stac
-import educe.util
-
-from ..features import\
-    players_for_doc,\
-    FeatureInput
-from ..addressee import guess_addressees
```

Comment:
Note that this may be a place where automated coverage checking is useful (sort of; if you have useless unit tests “covering” the code, you're still carrying the baggage around).  Coverage checking can of course also be misleading in the positive case, but in the negative is sometimes worth paying attention to.